django/applications/catmaid/control/sampler.py

```diff
@@ -963,15 +965,15 @@ def add_multiple_sampler_domains(request, project_id, sampler_id):
             "ends": [{
                 "id": e.id,
                 "node_id": e.end_node_id
-            } for e in domain_ends]
+            } for e in domain_ends] # FIXME: domain_ends is not defined in this function
         })
 
     return JsonResponse(result_domains, safe=False)
 
 
 @api_view(['GET'])
 @requires_user_role([UserRole.Browse])
-def list_domain_intervals(request, project_id, domain_id):
+def list_domain_intervals(request, project_id, domain_id): # FIXME: list_domain_intervals is defined twice
```

Comment:
I wasn't aware this would even "compile"

Reply:
It's surprising this function is defined twice. Let's remove this implementation of `list_domain_intervals()` and let's keep the one on L1290.